server/vertx-server/src/main/scala/sttp/tapir/server/vertx/streams/ReadStreamCompatible.scala

```diff
@@ -20,3 +21,10 @@ object ReadStreamCompatible {
     override def fromReadStream(s: ReadStream[Buffer]): Nothing = ???
```

Comment:
remove incompatible - not needed anymore

Reply:
done